### PR TITLE
[1LP][RFR] pod upload scripts have to be updated to support 5.9

### PR DESCRIPTION
**Pull request by izapolsk**

### CODE REVIEW

File: scripts/template_upload_all.py [hunk 1]
```diff
@@ -294,7 +294,10 @@ def browse_directory(dir_url):
         if key == 'template_upload_openshift':
             # this is necessary because headers don't contain last-modified date for folders
             # todo: remove this along with refactoring script
-            url = name_dict[key] + 'cfme-template.yaml'
+            if '5.8' in name_dict[key]:
+                url = name_dict[key] + 'cfme-template.yaml'
+            else:
+                url = name_dict[key] + 'templates/' + 'cfme-template.yaml'
```
Comment (rlbabyuk) [inline scripts/template_upload_all.py]: **Question:** This covers also 5.7 or just 5.9?

Comment (izapolsk) [reply scripts/template_upload_all.py]: podified cfme exists from 5.8+

Comment (mfalesni) [reply scripts/template_upload_all.py]: **REQUIRED**: `'templates/cfme-template.yaml'`

**Comment by izapolsk** [comment]
@mfalesni , all those scripts are written in such style. url.join should be used there instead. However, I don't want to rewrite all such code occurrences. I just want to follow scripts' style.
Are you sure it makes sense to change one row ?

btw, I have a card to rewrite all those scripts.

**Comment by izapolsk** [comment]
@mfalesni, as you noticed about I don't share your opinion on that but I fixed what you want.

**Comment by mfalesni** [comment]
@izapolsk hehe, I didn't actually mean using urljoin, just not having to join two string constants, but I guess this is good as well :)

**Pull request merged**